Gems/Georeferencing/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+
+o3de_gem_setup("Georeferencing")
```

Comment:
Nit: shouldn't we call it something like `Georeference`, `LevelGeoreference` or `Georeferencer`? (the same why we call gems `Spawner` and not `Spawning`)

Reply:
I decided to use this name since it is technical term : https://en.wikipedia.org/wiki/Georeferencing
```
Georeferencing or georegistration is a type of coordinate transformation that binds a digital raster image or vector database that represents a geographic space (usually a scanned map or aerial photograph) to a spatial reference system, thus locating the digital data in the real world.[1][2] It is thus the geographic form of image registration or image rectification. The term can refer to the mathematical formulas used to perform the transformation, the metadata stored alongside or within the image file to specify the transformation, or the process of manually or automatically aligning the image to the real world to create such metadata.
```